zokrates_book/src/reference/ir.md

```diff
@@ -0,0 +1,17 @@
+# ZIR
+
+ZIR is the intermediate representation ZoKrates uses to represent programs. It is close to R1CS but still encapsulates witness generation.
+
+**Note that ZIR is still in development and can change without notice.**
+
+When generating R1CS constraints, very large numbers are often used, which can make reading ZIR hard for humans.
+To mitigate this, ZIR uses an abuse of language (or an isomorphism) when displaying field elements: they are shown as members of the interval `[-(p-1)/2, (p-1)/2]`.
```

Comment:
I think abuse of language is a bit strong maybe :D 
I propose: Displays field elements by using the following isomorphism:
field element x with values [0,(p-1)/2] as x
field element x with values [(p-1)/2+1,p-1] as -p+x